### ISSUE TITLE
Secondary grey text colour is not accessible

### ISSUE THREAD

**Issue by ghost** [DivanteLtd/next#389]
Secondary grey text colour is not accessible

The font colour #a3a5ad used e.g. on category pages for Sort By, Filters, Products found copy does not pass WCAG accessibility guidelines for small text on a white background. https://monsido.com/tools/contrast-checker

There are many areas of the site that fail an automated accessibility scan, contrast, redundant text duplication etc https://wave.webaim.org/report#/https://vsf-next-demo.storefrontcloud.io/c/men some things are not shown due to issues with server rendering

**Comment by Qrzy** [comment]
Reported to SFUI team: https://github.com/DivanteLtd/storefront-ui/issues/1065

**Comment by mbaran-dvnt** [comment]
Hi guys,

I've gone through all of the designs, fixed the grey colors in the style guide and I hope I've changed every place with wrong grey on the pages. I've also fixed some of the font size issues (text button). All should be now aligned with a11y :)

**Comment by pspaczek** [comment]
https://github.com/DivanteLtd/storefront-ui/pull/1121 after 0.7.5 release it will be fixed :)
